src/style.css

```
@@ -1,3 +1,86 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
 body {
   margin: 0;
+  --color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  min-width: 1024;
+
```

Use width: 100% instead of min and max
```suggestion
  width: 100%;
```